.appveyor.yml

```diff
@@ -124,7 +125,7 @@ install:
 build_script:
     - mkdir build
     - cd build
-    - '"c:/Program Files (x86)/Microsoft Visual Studio 14.0/VC/vcvarsall" %vc_arch%'
+    - '"C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build\vcvars%ssl_arch:~3%.bat"'
```

Comment:
Let's not mess with variables this way. What's wrong with `%vc_arch%`?
The matrix has different _arch variables because they have different formats.

Reply:
I just follow appveyor doc. There are only these two .bat files, no more .bat accepting arch argument. (I checked in rdp session)

Reply:
But you are still abusing `ssl_arch` for non-SSL related stuff. After this change `vc_arch` is not used anymore.
What about changing values of `vc_arch` to match its new format?